web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/UtilsApiController.java

```diff
@@ -114,6 +124,12 @@
                     clinicalVariant.getLevel(),
                     MainUtils.listToString(new ArrayList<>(clinicalVariant.getDrug()), ", "),
                     MainUtils.listToString(new ArrayList<>(clinicalVariant.getDrugPmids()), ", ")));
+                Set<ArticleAbstract> articleAbstracts = clinicalVariant.getDrugAbstracts();
+                List<String> abstracts = new ArrayList<>();
+                for (ArticleAbstract articleAbstract : articleAbstracts) {
+                    abstracts.add(articleAbstract.getAbstractContent());
+                }
+                MainUtils.listToString(abstracts, "; ");
```

Comment:
Did you forget to assign this to something? Because you are not using the returned string for anything.